### PR TITLE
.gitignore update

### PR DESCRIPTION
**Pull request by Ofelia1870**

### CODE REVIEW

**Review by rkutsel — COMMENTED** [review]
Hey this is not fully done.

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (rkutsel) [inline .gitignore:1]: this line does not belong here

**Review by rkutsel — APPROVED** [review]

**Review by rkutsel — CHANGES_REQUESTED** [review]
test

**Review by Loggamon — APPROVED** [review]
This looks fine!

**Review by Loggamon — APPROVED** [review]
You're right

**Review by rkutsel — APPROVED** [review]

**Pull request merged**